### PR TITLE
Navbar!

### PR DESCRIPTION
**Pull request by c1jb**
Added all navbar functionality and validated existing backend code. Cleaned up other pages and took away the home button as it was unnecessary.

### CODE REVIEW

**Review by tbryant-tbryant — APPROVED** [review]
Everything looks good, just some refactoring and removal of extra code. Good to merge.

File: FrontEnd/myapp/src/App.js [hunk 1]
```diff
@@ -1,157 +1,18 @@
...
-import { Portfolio } from "./Portfolio.js";
-import { useCookies } from "react-cookie";
-import { Dynamic } from "./Dynamic.js";
 
-import { useEffect } from "react";
-import { componentDidMount } from "react";
-import axios from "axios";
 require("dotenv").config();
 
 function App() {
```
Comment (rayangendre) [inline FrontEnd/myapp/src/App.js:12]: I like how we now have a clean App.js, makes the base running easier and more class based

File: FrontEnd/myapp/tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
```
Comment (rayangendre) [inline FrontEnd/myapp/tailwind.config.js:1]: would like some explanation of this

**Review by rayangendre — APPROVED** [review]
Overall good changes to the App.js, some stuff will need explaining on Sunday

File: FrontEnd/myapp/tailwind.config.js (hunk 2 above)
Comment (c1jb) [reply FrontEnd/myapp/tailwind.config.js:1]: good catch, I am updating with a deleted version of this. Originally, I was going to use tailwind as our navbar service but found the default boostrap one easier. Thank you.

**Pull request merged**